### DISCUSSION TITLE
Question about converting waymo to kitti format

### DISCUSSION THREAD

**Discussion by JiSuanJiDaWang** [open-mmlab/mmdetection3d#2037] [Q&A] [answered]
I used num_works 4 to convert the waymo to kiiti format. But the speed is surprisingly slow. Is that reasonable?
![img_v2_9e13d7b9-6ec5-4b79-bcbb-6f2fff5335cg](https://user-images.githubusercontent.com/82489384/202633048-d58a2e00-e677-440e-b13a-9f5ba46fef8f.png)

**Comment by JingweiZhang12** [comment, marked as answer]
It's a little slower. Maybe you can try larger `num_workers`.

**Reply by JiSuanJiDaWang** [reply]
Hi, I tried 20 workers instead, but it still shows 0.0 tasks/s. And I also noticed that the thread created by the program is not always CPU-consuming. Is it normal?
